docs/source/aggregation.rst

```diff
@@ -0,0 +1,172 @@
+.. _aggregation:
+
+===========
+Aggregation
+===========
+
+This chapter provides information about the passing aggregates of jobs to operation functions.
+
+
+.. _aggregator_definition:
+
+aggregator
+==========
+
+An :class:`~flow.aggregator` is used as a decorator for an operation function which acts as an entry point to define the type of aggregation to perform while submitting or running the operation function.
+
+.. code-block:: python
+
+    # project.py
+    from flow import FlowProject, aggregator
+
+    class Project(FlowProject):
+        pass
+
+    @aggregator()
+    @Project.operation
+    def op1(*jobs):
+        pass
+
+    @Project.operation
+    def op2(job):
+        pass
+
+    if __name__ == '__main__':
+        Project().main()
+
+By default, if :class:`~flow.aggregator` is used as a decorator, aggregate of all the jobs present in the project will be created.
+In the above example, ``op1`` can be referred to as an *aggregate operation* where all the jobs present in the project are passed as arbitraty arguments (or ``*args``) and ``op2`` is a *normal operation* where only a single job is passed as a parameter.
```

Comment:
I think mentioning the fact that all the functions (not sure if all, but definitely most?) used by the operation, such as functions of post and pre conditions, now require passing of `*jobs` instead of a `job`.

Reply:
Tried resolving here: https://github.com/glotzerlab/signac-docs/pull/122/files#diff-d053cd842dd6fe746188c4339762f7e336b763b94d513473d65900b9836a09f0R41